src/@context/Asset.tsx

```diff
@@ -83,19 +83,6 @@ function AssetProvider({
     setLoading(false)
   }
 
-  const setPurgatory = useCallback(async (did: string): Promise<void> => {
-    if (!did) return
-
-    try {
-      const result = await getAssetPurgatoryData(did)
```

Comment:
as you see with above GitHub warnings, `getAssetPurgatoryData()` is now unused in this file, so:

1. the import needs to be removed
2. the method itself might not be needed anymore, if we don't use it anywhere else then it can be removed